SU-HSDO/suhumsci
# SHS-3750: Breakout Box does not span the full width

# READY FOR REVIEW

## Summary
Make elements with `hs-breakout-box` full width

## Need Review By (Date)
02/09

## Urgency
medium

## Steps to Test
1. Create a Flexible page. In the text area, add some paragraphs with different lengths (at least one of them should be shorter than a full line, just some words).
2. Apply the "Breakout Box" style to some of the paragraphs, using the CKEditor "Style" selector.
3. Make sure that the Breakout Box background spans the full editor width, even when the text is shorter than a line.
4. Save the node. Confirm that the Breakout Box background spans the full container width, even when the text is shorter than a line.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
